rdkit/Dbase/DbReport.py

```diff
@@ -30,7 +32,9 @@
   from rdkit.Dbase.DbConnection import DbConnect
   from rdkit.Dbase import DbInfo
   from rdkit.Reports.PDFReport import PDFReport, ReportUtils
-  import os, tempfile, sys
+  from reportlab.lib import colors
```

Comment:
This is all in the `else` block of the try/import/except/else statements at the beginning, so will not be excecuted if reportlab is missing. A few lines down is actually a duplicated import of `reportlab.lib.units.inch`.